### PR TITLE
Fix several `--help` command related issues

### PR DESCRIPTION
**Pull request by iamrasel**

### CODE REVIEW

File: src/fastfetch.c [hunk 1]
```diff
@@ -522,13 +524,13 @@ static inline void printCommandHelp(const char* command)
...
     {
-        constructAndPrintCommandHelpFormat("gamepad", "{1}", 1,
+        constructAndPrintCommandHelpFormat("gamepad", "{1}", 2,
             "Name",
             "Identifier"
         );
     }
     else
-        fprintf(stderr, "No specific help for command %s provided\n", command);
+        fprintf(stderr, "No specific help for command \'%s\' provided\n", command);
```
Comment (CarterLi) [inline src/fastfetch.c:533]: I don't think we need `\` here

File: src/fastfetch.c [hunk 2]
```diff
@@ -439,7 +441,7 @@ static inline void printCommandHelp(const char* command)
             "Album name"
         );
     }
-    else if(ffStrEqualsIgnCase(command, "datetime-format") == 0 || ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format"))
+    else if(ffStrEqualsIgnCase(command, "datetime-format") || ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format"))
```
Comment (CarterLi) [inline src/fastfetch.c:444]: We can remove `|| ffStrEqualsIgnCase(command, "date-format") || ffStrEqualsIgnCase(command, "time-format")` as the modules `date` and `time` no longer exist.

**Comment by CarterLi** [comment]
Thanks for the fix! Because they are not in module implementation files, I keep forgetting to update them.

I actually considered moving them into `module.c`, so that we can reuse the macros `FF_*_NUM_FORMAT_ARGS`, but I also forgot it.

**Comment by iamrasel** [comment]
Just committed with suggested removals, please recheck.

**Pull request merged**